src/haddock/modules/scoring/sasascore/__init__.py

```diff
@@ -0,0 +1,143 @@
+"""
+Surface accessibility scoring module.
+
+This module performs a solvent accessibility analysis based on 
+some user-defined residues that should be buried or accessible.
+
+If a supposedly buried (resp. accessible) residue is accessible (resp. buried),
+the score should increase by one. The lower the final score the more consistent
+the model with the user data.
```

Comment:
Do we want to add here how to use the expendable parameters ?
resdic_accessible_X = []
resdic_buried_X = []

Reply:
good point!